### ISSUE TITLE
Brew Package

### ISSUE THREAD

**Issue by cpb8010** [thejandroman/bing-wallpaper#14]
Have you thought about publishing this as a homebrew package for even easier install and updates?

Would you mind if someone else did?

**Comment by thejandroman** [comment]
@cpb8010, though I hadn't considered it I'm sure if you find value in it others will as well. Go for it! 😁

**Comment by nyarasha** [comment]
That would be awesome! @cpb8010 did you end up making a homebrew package? If not, I might go ahead

**Comment by cpb8010** [comment]
I didn't! It's been on the bottom of my list :(

**Comment by nyarasha** [comment]
Okay, I took a first pass at trying to make a brew tap and package - README and tap is up here: https://github.com/nyarasha/homebrew-bing-wallpaper

Not sure how to assign myself this issue or whether someone else wants to take it up, but it's at least started.

What else do we want the installer to do, other than installing the files on the user's computer in a folder? Deal with the plist?

**Comment by cpb8010** [comment]
I added mine to cron and have it run hourly, but maybe that's too much as a default.
https://github.com/cpb8010/bing-wallpaper/commit/8a1bf0587199c67f78ff9086391dd9cda297edc8

```
@hourly /Users/username/Code/bing-wallpaper/bing-wallpaper.sh -w >/dev/null 2>&1
```

**Comment by nyarasha** [comment]
Yeah, I was trying to figure out how to make that user-settable or opt-in; mine runs every 10 hours via a .plist file (on Mac OSX). Any suggestions?

**Comment by cpb8010** [comment]
I don't know homebrew enough to find any good examples of user input, so I'd just set a default and log some directions for manual changes.

**Comment by laggardkernel** [comment]
Here's the formula you want with a service, https://github.com/laggardkernel/homebrew-tap. Cause the author of this project doesn't do a regular release, only installation from `HEAD` is available.

```shell
brew install --HEAD laggardkernel/tap/bing-wallpaper
```

The formula installs the script as `bing-wallpaper`, and a service could be enable to download pictures,  and set them as wallpaper every 4 hours.

```shell
brew services start bing-wallpaper
```

AFAIK, there's no way to let user configure the service `.plist`, if you wanna change the download location or do other customizations. Modify the `homebrew.mxcl.bing-wallpaper.plist` after the service is added, then restart the service to make it take effect.
